lib/lighthouse/auth/client_credentials/service.rb

```diff
@@ -37,6 +37,7 @@ def initialize(token_url, api_scopes, client_id, aud_claim_url, rsa_key, service
       #
       # @return [String] the access token needed to make requests
       #
+
```

Comment:
Can your remove this newline change. This way this file won't show up in the changed files list and you won't have to involve lighthouse approvals for this PR

Reply:
newline has been removed